energyplus_regressions/diffs/table_diff.py

```diff
@@ -435,8 +484,19 @@ def table_diff(thresh_dict, inputfile1, inputfile2, abs_diff_file, rel_diff_file
                                table_not_in_1, table_not_in_2)
             continue
 
+        # create a list of order-dependent table uheading keys, tables that include these keys in the name
+        # these will use strict row order enforcement
+        row_order_dependent_table_keys = ['Monthly', 'Topology']
+
+        # always process the first table into a base hdict
         hdict1, horder1 = table2hdict_horder(table1)
-        hdict2, horder2 = table2hdict_horder(table2)
+
+        # if we are in a row order dependent table, don't pass table1 as a baseline, just use the literal in-place order
+        if any(k in uheading1 for k in row_order_dependent_table_keys):
+            hdict2, horder2 = table2hdict_horder(table2)
+        # but for all other tables, we can use the first table as a baseline to carefully match up the rows
+        else:
+            hdict2, horder2 = table2hdict_horder(table2, table1)
```

Comment:
And this block will either pass in table2 alone, in which case order dependence will be respected, or pass table1 as an optional second parameter.  In that case, table1 order will be used to try to detect row matching, and allow the tables to pass if it's purely a reorder.